samples/VS2015/SpecFlow 1.9.0/net461 1.0.0/Sample.Website.Tests/project.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": "1.0.0-*",
+  "description": "Sample.Website.Tests Class Library",
+  "authors": [ "stajs" ],
+  "packOptions": {
+    "tags": [ "" ],
+    "projectUrl": "",
+    "licenseUrl": ""
+
+  },
+
+  "frameworks": {
+    "net461": {}
+  },
+
+  "dependencies": {
+    "Sample.Website": "1.0.0-*",
+    "SpecFlow": "2.1.0",
+    "xunit": "2.2.0-beta2-build3300",
+    "dotnet-test-xunit": "2.2.0-*",
+  },
+
+  "tools": {
+    "dotnet-specflow": "1.0.0-beta4"
```

Comment:
We're now a dotnet tool, so you need to list the dependency in the tool-part of project.json.